python/ray/data/grouped_dataset.py

```diff
@@ -98,22 +92,23 @@ def aggregate(self, *aggs: Tuple[AggregateFn]) -> Dataset[U]:
         map_results = np.empty((num_mappers, num_reducers), dtype=object)
         for i, block in enumerate(blocks):
             map_results[i, :] = partition_and_combine_block.remote(
-                block, boundaries, self._key, agg)
+                block, boundaries, self._key, *aggs)
```

Comment:
nit: here I think we can just pass the entire tuple as one argument, instead of unpacking it?